$NetBSD: patch-aa,v 1.1.1.1 2005/12/04 23:40:59 shattered Exp $

Proposed fix for SF bug 1113391 (Dictionary strings not encoded in UTF-8)

--- Dictionary.rb.orig	2004-12-08 02:07:47.000000000 +0000
+++ Dictionary.rb
@@ -37,7 +37,7 @@ class DictionaryString < String
 
     def write(data)
         @pos = data.pos
-        data.putstring(self)
+        data.putstring(self.unpack("C*").pack("U*"))
     end
 
     include Pointable
